release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -3748,3 +3748,28 @@ $ oc adm release info 4.10.47 --pullspecs
 ==== Updating
 
 To update an existing {product-title} 4.10 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster within a minor version by using the CLI] for instructions.
+
+[id="ocp-4-10-50"]
+=== RHSA-2023:0241 - {product-title} 4.10.50 bug fix and security update
+
+Issued: 2023-01-24
+
+{product-title} release 4.10.50, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2023:0241[RHSA-2023:0241] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:0240[RHBA-2023:0240] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.10.50 --pullspecs
+----
+[id="ocp-4-10-50-bug-fixes"]
+==== Bug fixes
+
+* Previously, when rotating {rh-openstack-first} credentials, the Cinder Container Storage Interface (CSI) driver wwould continue to use old credentials. Using any old credentials that were invalid would cause all volume operations to fail. With this update the Cinder CSI driver is updated automatically when the {rh-openstack-first} credentials are updated. (link:https://issues.redhat.com/browse/OCPBUGS-4717[*OCPBUGS-4717*])
```

Comment:
```suggestion
* Previously, when rotating {rh-openstack-first} credentials, the Cinder Container Storage Interface (CSI) driver would continue to use old credentials. Using any old credentials that were invalid would cause all volume operations to fail. With this update, the Cinder CSI driver is updated automatically when the {rh-openstack-first} credentials are updated. (link:https://issues.redhat.com/browse/OCPBUGS-4717[*OCPBUGS-4717*])
```